### ISSUE TITLE
"glslangValidator -V" doesn't #define VULKAN

### ISSUE THREAD

**Issue by Tulon** [KhronosGroup/glslang#254]
Trying to compile the following no-op vertex shader:
```
#version 430

void main()
{
#ifdef VULKAN
#  error "VULKAN defined"
#else
#  error "VULKAN not defined"
#endif
}
```
with `glslangValidator -V /path/to/shader.vert` produces:
```
Warning, version 430 is not yet complete; most version-specific features are present, but some are missing.
ERROR: 0:8: '#error' : VULKAN not defined
ERROR: 1 compilation errors.  No code generated.



Linked vertex stage:


SPIR-V is not generated for failed compile or link
```

According to [GL_KHR_vulkan_glsl](https://www.khronos.org/registry/vulkan/specs/misc/GL_KHR_vulkan_glsl.txt), an implicit `#define VULKAN 100` has to be provided by the compiler.

I am using glslangValidator from LunarG Vulkan SDK version 1.0.11.0

**Comment by johnkslang** [comment]
Agreed there is a missing `#define VULKAN 100`, which needs to be added.

**Comment by johnkslang** [comment]
This is fixed by commit e512cd943ed25174273f7520bd32346f4659597b.
